### ISSUE TITLE
OMP_SET_NESTED is deprecated - would be nice to replace with equivalent newer variant

### ISSUE THREAD

**Issue by hppritcha** [lanl/SNAP#18]
We are getting these warnings from a compiler (either intel oneAPI or gnu not sure):
```
OMP: Info #274: omp_set_nested routine deprecated, please use omp_set_max_active_levels
```
when compiling snap.  It would be nice to update snap to use the newer omp nested directive.

**Comment by zerr** [comment]
Will do

**Comment by zerr** [comment]
Yikes. Forgot to say this was resolved with v1.12.
